controllers/horizon_controller.go

```diff
@@ -547,3 +579,52 @@ func (r *HorizonReconciler) ensureHorizonSecret(
 
 	return nil
 }
+
+func (r *HorizonReconciler) renderMemcached(instance *horizonv1alpha1.Horizon) *memcachedv1.Memcached {
+	return &memcachedv1.Memcached{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "memcached.openstack.org/v1beta1",
+			Kind:       "Memcached",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-memcached", instance.Name),
```

Comment:
we probably not need to postfix the name of each Memcached instance with "-memcached" the Name only need to be unique per namespace per Kind.

Reply:
It needs to have a different name compared to the instance.Name though, so I figured this makes sense. If I try to call it instance.Name, then the memcached CR returns the following error:
```
❯ oc get memcached horizon -o jsonpath='{.status.conditions[3].message}'
Service config create error occured error create/updating configmap: Object openstack/horizon-config-data is already owned by another Horizon controller horizon
```

Reply:
Hm probably Memcached reconciler generates its own config map name based on just the instance.Name + "-config-data". I would say this is a problem in the Memcached implementation. I think Memcached should generate its config map name based on instance.Kind + instance.Name + "-config-data". That way we could avoid the need to each creator of a Memcached add the Kind to the name of the Memcached instance.